apex/amp/_initialize.py

```diff
@@ -139,11 +142,10 @@ def __call__(self, module, state_dict, prefix, local_metadata):
 
 
 def _initialize(models, optimizers, properties, num_losses=1, cast_model_outputs=None):
-    from apex.parallel import DistributedDataParallel as apex_DDP
     from .amp import init as amp_init
 
     optimizers_was_list = False
-    if isinstance(optimizers, torch.optim.Optimizer) or isinstance(optimizers, LARC):
+    if isinstance(optimizers, torch.optim.Optimizer) or ('LARC' in sys.modules and isinstance(optimizers, LARC)):
```

Comment:
This line does not work in linux, because 'LARC' is always not in sys.modules. Please revert this commit. 

```
>>> import sys
>>> import apex
>>> 'LARC' in sys.modules
False
>>> 'apex.parallel.LARC' in sys.modules
True
```
```

Reply:
Hm, perhaps @ptrblck can comment on this. I am not sure what that change is supposed to signify. But from the looks of it, you're right that it should be `'apex.parallel.LARC' in sys.modules`.